app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_tuple = tuple(command.split(" "))
```

Comment:
no need to use space inside split method, it uses space by default

Reply:
without space inside split method "ValueError: empty separator" is beeing rised